.github/workflows/tox.yml

```diff
@@ -9,17 +9,18 @@ jobs:
         matrix:
           environment: [pep8,py3]
           python-minor-version: [6,8,10]
-          is-zed: 
+          is-not-python36:
             - ${{ (github.base_ref == 'stackhpc/zed') || (github.ref == 'refs/heads/stackhpc/zed')  }}
+            - ${{ (github.base_ref == 'stackhpc/2023.1') || (github.ref == 'refs/heads/stackhpc/2023.1')  }}
```

Comment:
Are these conditions ANDed or ORed together? If AND, it will never be true

Reply:
Related discussion - https://github.com/orgs/community/discussions/25641

Reply:
Corrected, will something like this work properly ?